lib/src/HttpAppFrameworkImpl.h

```diff
@@ -392,6 +392,16 @@ class HttpAppFrameworkImpl : public HttpAppFramework
         return false;
     }
 
+    virtual size_t getCurrentThreadIndex() const override
+    {
+        auto *loop = trantor::EventLoop::getEventLoopOfCurrentThread();
+        if (loop)
+        {
+            return loop->index();
+        }
+        return -1;
```

Comment:
This might be overkill, but maybe use `std::numeric_limits<size_t>::max()` to make it more clear what is happening?